packages/api/endpoints/granules.js

```diff
@@ -300,6 +302,7 @@ async function put(req, res) {
 
   if (action === 'move') {
     // FUTURE - this should be removed from the granule model
+    // TODO -- Phase 3 -- This needs to be pulled out of the granule model
```

Comment:
This doesn't strictly need to be done here, so adding a comment as scope limit.

Reply:
> You're saying getFilesExistingAtLocation needs to be moved because we're trying to eliminate the granuleModel?

Yes

> I haven't done any of these "Remove Dynamo" tickets. Is that out of scope? Are we taking another pass at this kind of thing in phase 4?

Not out of scope for P3, out of scope for 2804.   I'm only making these updates because of common code as relates to the bulk granules endpoint logic.  

In this case, it falls in the CUMULUS-2804 cleanup, and there's a PR (dependent on this one) https://github.com/nasa/cumulus/pull/2958/files